README.md

```diff
@@ -34,24 +35,32 @@ To get started with Scaffold-ETH 2, follow the steps below:
 ```
 git clone https://github.com/scaffold-eth/scaffold-eth-2.git
 cd scaffold-eth-2
-yarn install
+yarn install && forge install --root packages/foundry
 ```
 
-2. Run a local network in the first terminal:
+2. Create `packages/foundry/.env` file based of on `packages/foundry/.env.example`
+
+```
+cp packages/foundry/.env.example packages/foundry/.env
+```
+
+> NOTE: If you are using windows and not using git bash or wsl, use `copy packages\foundry\.env.example packages\foundry\.env`
+
+3. Run a local network in the first terminal:
```

Comment:
I saw that you already commented about this in the PR comment. Having to do this is a PITA (the 3 magic commands don't work out of the box), so I've been trying some options (postinstall as you mentioned)

branch: https://github.com/scaffold-eth/scaffold-eth-2/commits/foundry-main--cp-env/
commit: https://github.com/scaffold-eth/scaffold-eth-2/commit/951f7336d9b33cf666a658c53c30387c0520e7c8

Using the smallest `copy` lib that I could find (even tho it installs a few other deps, maybe we can explore other options), which should work on any OS.

I remember that we had some issues with the postinstall script (not running at all in some OS? Or running after every time you run `yarn install`? Added the `--no-overwrite` in case it's the latter).

What do you think?

Reply:
If we take this path, this can help to decide about the lib to use. https://pkg-size.dev/

eg. https://pkg-size.dev/cpy-cli => 1.3MB vs the 5 KB advertised (which doesn't count the deps)

Reply:
Was just trying https://github.com/scaffold-eth/scaffold-eth-2/tree/foundry-main--cp-env and found a minor disadvantage that: 

1. First you clone above repo and `yarn install` this creates `.env.example` nicely in `pacakge/foundry`
2. If you try to install node dependency in `foundry` package `cd packages/foundry` and `yarn add viem` it will fail : 
![Screenshot 2024-06-26 at 2 31 44 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/12f08df4-bc72-427c-9ec8-5bc82a65316a)

basically you won't be able to install any node / js dependency in `foundry` package because it will fail since we are using `---no-overwrite` it throwing error

An alternatie solution will be finding a library which doesn't throw error or writting our own node script which doesn't throw error if `.env` is already present

Reply:
I think this is what we need: https://www.npmjs.com/package/shx (we can use commands from linux, and we could copy... or even test if the file exist before copying)

And it takes less space https://pkg-size.dev/shx

I'll give it a shot

Reply:
What about this (using shx): https://github.com/scaffold-eth/scaffold-eth-2/commit/46443fb6edc9f2cd08ab0f0b156c917b101a4747

I pushed it to https://github.com/scaffold-eth/scaffold-eth-2/commits/foundry-main--cp-env/

If we like it, we can have Pablo to test on Windows.

Reply:
Noiicee, just tried it out and works great!! Tysm!!, I think this works great!! Will merge in this changes!